syscalls.go

```diff
@@ -145,6 +145,13 @@ var haveNoPreallocMaps = internal.NewFeatureTest("prealloc maps", "4.6", func()
 	return nil
 })
 
+func wrapMapLookupError(err error) error {
```

Comment:
Nit: it'd be ok to inline this into `Map.lookup` I think.

Reply:
Done.